### ISSUE TITLE
pip install open3d doesn't work without additional system dependencies

### ISSUE THREAD

**Issue by meder411** [isl-org/Open3D#2362]
**IMPORTANT: Please use the following template to report the bug.**
_____

**Describe the bug**
Most likely just a documentation issue, but simply running `pip3 install open3d` [as described in the docs](http://www.open3d.org/docs/release/getting_started.html#pip-pypi) does not work. You still need `libgl1-mesa-glx` installed.

Running in a docker container gives this output:

```
root@209d7cfb5cdf:/# python3
Python 3.8.2 (default, Jul 16 2020, 14:00:26) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import open3d
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/dist-packages/open3d/__init__.py", line 47, in <module>
    from open3d.open3d_pybind import camera
ImportError: libGL.so.1: cannot open shared object file: No such file or directory
```

**To Reproduce**

Build this Dockerfile:

```
FROM nvidia/cuda:latest

RUN apt-get update && apt-get install -y \
    python3-pip \
    && rm -rf /var/lib/apt/lists/*

RUN pip3 install open3d
```

Run the container and then try:

```
>> python3
>> import open3d
```

**Expected behavior**
The documentation suggests I should have complete functionality with the `pip` installation, so I would expect as much.

**Screenshots**
--

**Environment (please complete the following information):**

- Operating system: (e.g. OSX 10.15, Ubuntu 18.04, Windows 10 64-bit) Ubuntu 20.04
- Python version: (e.g. Python 3.6) 3.8.2
- Open3D version: whichever is available via `pip`
- Is this remote workstation?: yes 
- How did you install Open3D?: (e.g. pip, conda, build from source) `pip`
- Compiler version (if built from source): (e.g. gcc 7.5, clang 7.0)

**Additional context**
Perhaps a dependency list or replacing the recently removed sample Dockerfile could be a quick and easy solution? Simply installing `libgl1-mesa-glx` resolved the import issue.

**Comment by yxlao** [comment]
@meder411 I think what you described is true. In desktop Ubuntu, `pip install open3d` will work out-of-the-box. We'll update the documentation for the Docker or server environment.

**Comment by ssheorey** [comment]
Minimal dockerfile is now included in the documentation since PR #3396.

http://www.open3d.org/docs/release/getting_started.html#docker
